benchmarks/run_naslib.py

```diff
@@ -0,0 +1,99 @@
+import argparse
+import os
+import subprocess
+
+
+def run(args: argparse.Namespace) -> None:
+    kurobako_cmd = os.path.join(args.path_to_kurobako, "kurobako")
+    subprocess.run(f"{kurobako_cmd} --version", shell=True)
+
+    os.makedirs(args.out_dir, exist_ok=True)
+    study_json_filename = os.path.join(args.out_dir, "studies.json")
+    subprocess.check_call(f"echo >| {study_json_filename}", shell=True)
+    solvers_filename = os.path.join(args.out_dir, "solvers.json")
+    subprocess.check_call(f"echo >| {solvers_filename}", shell=True)
+    problems_filename = os.path.join(args.out_dir, "problems.json")
+    subprocess.check_call(f"echo >| {problems_filename}", shell=True)
+
+    searchspace_datasets = [
+        "nasbench201 cifar10",
+        "nasbench201 cifar100",
+        "nasbench201 ImageNet16-120",
+    ]
+
+    for searchspace_dataset in searchspace_datasets:
+
+        python_command = f"benchmarks/naslib/problem.py {searchspace_dataset}"
+        cmd = (
+            f"{kurobako_cmd} problem command python3 {python_command}"
+            f"| tee -a {problems_filename}"
+        )
+        subprocess.run(cmd, shell=True)
```

Comment:
Could you tell me why we call `problem.py` in a subprocess? I guess we can call functions and classes in `problem.py` if we import it.

Reply:
ditto.
https://github.com/optuna/optuna/pull/3465/files#r849252283